.github/workflows/rename-module.yml

```
@@ -17,71 +21,96 @@ jobs:
         with:
           fetch-depth: 0 # everything
 
+      - run: git fetch --tags https://github.com/ethereum/go-ethereum.git
+
+      - run: git checkout ${{ inputs.source }}
+
+      - name: References pointing to source
+        # NOTE: This step assumes that the source has been checked out, which
+        # might not hold if reordered.
+        run: |
+          git branch --points-at HEAD;
+          git tag --points-at HEAD;
```

supernit I don't think semi colons are necessary

```suggestion
          git branch --points-at HEAD
          git tag --points-at HEAD
```